srcpkgs/python3-codespell/template

```diff
@@ -1,20 +1,20 @@
 # Template file for 'python3-codespell'
 pkgname=python3-codespell
-version=2.1.0
-revision=2
+version=2.2.2
+revision=1
 build_style=python3-module
-hostmakedepends="python3-setuptools"
+hostmakedepends="python3-setuptools_scm python3-wheel"
 depends="python3-chardet"
-checkdepends="python3-pytest-cov $depends"
+checkdepends="${depends} python3-pytest"
 short_desc="Checks code for common spelling mistakes (Python3)"
-maintainer="Orphaned <orphan@voidlinux.org>"
-license="GPL-2.0-or-later"
-homepage="https://github.com/codespell-project/codespell"
-distfiles="https://github.com/codespell-project/codespell/archive/v${version}.tar.gz"
-checksum=8d0730b6ff35e56bda8d9600ccffbf27bb83905600775bf869cb8b9936b108c0
+maintainer="icp <pangolin@vivaldi.net>"
+license="GPL-2.0-only"
+homepage="https://pypi.org/project/codespell/"
+distfiles="${PYPI_SITE}/c/codespell/codespell-${version}.tar.gz"
+checksum=c4d00c02b5a2a55661f00d5b4b3b5a710fa803ced9a9d7e45438268b099c319c
 conflicts="python-codespell>=0"
 
 pre_check() {
-	# Remove test that needs itself
-	vsed -i '/run_codespell/,/==4/d' codespell_lib/tests/test_basic.py
+	vsed -e '/addopts/d' -i setup.cfg
+	export PATH="${wrksrc}//bin:${PATH}"
```

Comment:
```suggestion
	export PATH="${wrksrc}/bin:${PATH}"
```
and this can be `make_check_pre="${wrksrc}/bin:${PATH}"`

Reply:
Yes, that is what I did initially: `make_check_pre="env PATH=${wrksrc}/bin:${PATH}"` (I think that's what you mean), but it didn't work. And as I had a `pre_check` block anyway so I didn't investigate further why it wasn't working (it still doesn't).